rector.php

```diff
@@ -0,0 +1,117 @@
+        __DIR__ . '/packages/zend-file-transfer',
+        __DIR__ . '/packages/zend-filter',
+        __DIR__ . '/packages/zend-form',
+        __DIR__ . '/packages/zend-gdata',
+        __DIR__ . '/packages/zend-http',
+        __DIR__ . '/packages/zend-json',
+        __DIR__ . '/packages/zend-layout',
+        __DIR__ . '/packages/zend-ldap',
+        __DIR__ . '/packages/zend-loader',
+        __DIR__ . '/packages/zend-locale',
+        __DIR__ . '/packages/zend-log',
+        __DIR__ . '/packages/zend-mail',
+        __DIR__ . '/packages/zend-markup',
+        __DIR__ . '/packages/zend-measure',
+        __DIR__ . '/packages/zend-memory',
+        __DIR__ . '/packages/zend-mime',
+        __DIR__ . '/packages/zend-mobile',
+        __DIR__ . '/packages/zend-navigation',
+        __DIR__ . '/packages/zend-oauth',
+        __DIR__ . '/packages/zend-openid',
+        __DIR__ . '/packages/zend-paginator',
+        __DIR__ . '/packages/zend-pdf',
+        __DIR__ . '/packages/zend-progressbar',
+        __DIR__ . '/packages/zend-queue',
+        __DIR__ . '/packages/zend-reflection',
+        __DIR__ . '/packages/zend-registry',
+        __DIR__ . '/packages/zend-rest',
+        __DIR__ . '/packages/zend-search',
+        __DIR__ . '/packages/zend-search-lucene',
+        __DIR__ . '/packages/zend-serializer',
+        __DIR__ . '/packages/zend-server',
+        __DIR__ . '/packages/zend-session',
+        __DIR__ . '/packages/zend-soap',
+        __DIR__ . '/packages/zend-stdlib',
+        __DIR__ . '/packages/zend-tag',
+        __DIR__ . '/packages/zend-text',
+        __DIR__ . '/packages/zend-timesync',
+        __DIR__ . '/packages/zend-tool',
+        __DIR__ . '/packages/zend-translate',
+        __DIR__ . '/packages/zend-uri',
+        __DIR__ . '/packages/zend-validate',
+        __DIR__ . '/packages/zend-version',
+        __DIR__ . '/packages/zend-view',
+        __DIR__ . '/packages/zend-wildfire',
+        __DIR__ . '/packages/zend-xml',
+        __DIR__ . '/packages/zend-xmlrpc',
+        __DIR__ . '/packages/zend-service',
+        __DIR__ . '/packages/zend-service-akismet',
+        __DIR__ . '/packages/zend-service-amazon',
+        __DIR__ . '/packages/zend-service-audioscrobbler',
+        __DIR__ . '/packages/zend-service-console',
+        __DIR__ . '/packages/zend-service-delicious',
+        __DIR__ . '/packages/zend-service-ebay',
+        __DIR__ . '/packages/zend-service-flickr',
+        __DIR__ . '/packages/zend-service-livedocx',
+        __DIR__ . '/packages/zend-service-rackspace',
+        __DIR__ . '/packages/zend-service-recaptcha',
+        __DIR__ . '/packages/zend-service-shorturl',
+        __DIR__ . '/packages/zend-service-slideshare',
+        __DIR__ . '/packages/zend-service-strikeiron',
+        __DIR__ . '/packages/zend-service-twitter',
+        __DIR__ . '/packages/zend-service-yahoo',
+
+        //__DIR__ . '/packages/zend-test',
+        //__DIR__ . '/packages/zend-service-windowsazure',
```

Comment:
why are these commented out?

Reply:
zend-test and zend-service-windowazure are commented out because rector ran into an undefined class. I manually reviewed these packages instead.

Reply:
please add such comment in that file

Reply:
Added comment to the script in the PR body